setup.py

```diff
@@ -72,17 +90,18 @@ def read(fname):
         content = fp.read()
     return content
 
+
 DEPS = [
     'future',
-    'boto>=2.38.0',
```

Comment:
I'm strongly against unpinning prod dependencies like that. I'm fine with dev dependencies unpinned like you did in the same commit. So why? I have 0 trust in upstream packages not introducing breaking changes, a fortiori in major version/rewrites. We can relax dependencies a little like it was done for tabulate (pip should have a specific operator for that, like any correct package manager, tsss..), but beyond that? Too scary.

Reply:
This was done tongue-in-check :slightly_smiling_face:

Agreed on both counts. On the other hand, as a library we need be careful not to put too stringent requirements that will hamper our consumers, so perhaps the diskcache and kubernetes pinnings should be relaxed a bit. What do you think?

Reply:
(And anyway Travis fails for unknown reasons)

Reply:
> On the other hand, as a library we need be careful not to put too stringent requirements that will hamper our consumers, so perhaps the diskcache and kubernetes pinnings should be relaxed a bit.

Sure! I'd be fine with `kubernetes>=3.0.0,<4.0.0` and `diskcache>=2.4.1,<3.0.0`, yay?

Travis: meh. :'(

Reply:
Re Travis: ah, yet another "time.sleep" based test that may deserve a xfail?

Reply:
Eh, we could even start with diskcache 2.9.0 :wink:
:+1: for kubernetes (until the day they break their semver-support claim)

Reply:
Hooray for upgrading diskcache \o/